README.md

```diff
@@ -10,23 +10,68 @@ code in this repository extends an integration layer provided by [DrupalExtensio
 [![Latest Stable Version](https://poser.pugx.org/drupal/tqextension/v/stable?format=flat-square)](https://packagist.org/packages/drupal/tqextension)
 [![License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](https://packagist.org/packages/drupal/tqextension)
 
+## Requirements
+
+* composer
+* behat
+
 ## Installation
 
-- `curl -sS https://getcomposer.org/installer | php`
-- `vim composer.json`
-```json
-{
-  "require": {
-    "drupal/tqextension": "~1.0"
-  },
-  "config": {
-    "bin-dir": "bin"
-  }
-}
-```
-- `composer install`
+To install TqExtension on an existing project:
+
+- `composer require drupal/tqextension`
+- Add contexts to Behat, see [Configuration](#configuration) below
+
+## Behat Quick Start Guide
```

Comment:
The `behat` directory in this repository can be confusing to people not very familiar with behat--if someone finds this repo looking for some additional contexts, they may think that the quick starter folder (which is basically what the `behat` folder is in this repo) is needed.

This clarifies the purpose of the `behat` directory in the project and moves the `cp -r ...` command out of the installation steps which won't apply to people *just* wanting to get the contexts.